src/software/thunderscope/robot_diagnostics/robot_view.py

```diff
@@ -37,11 +38,22 @@ def __init__(self, load_fullsystem):
             RobotInfo(id, load_fullsystem, self.control_mode_signal)
             for id in range(MAX_ROBOT_IDS_PER_SIDE)
         ]
+        self.robot_status_widgets = [
+            RobotStatusView() for id in range(MAX_ROBOT_IDS_PER_SIDE)
+        ]
```

Comment:
So I thought it would be better to make them seperate for readability, plus it makes it easier to hide and show them. The hide and show methods are from QWidget, so I would have to have a child widget anyway if I put in directly in RobotInfo. So I just made a new widget instead

Reply:
I think it's okay to have a child widget inside of `RobotInfo` and have the implementation separately in another file. The main reason I brought this up was because the robot status seems fitting as a part of the robot info widget, alongside the other widgets youve added. But I'll leave it up to you if you think this approach is simpler.

Reply:
I see yea, I think the best way would be to keep the 2 widgets seperate, but have a third container widget which is all the info for 1 robot. Then, the robot_view widget is a scrollable collection of these. I named this RobotViewComponent and put this in robot_view instead of making a new file since it's pretty small and simple